addons/voice/org.openhab.voice.pico2wave/OSGI-INF/org.openhab.voice.pico2wave.internal.Pico2WaveService.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please put a `.gitignore` file into the `OSGI-INF` directory with content `/*.xml` and remove this auto-generated file.

Reply:
Now I understand what you mean, sorry for this delay :)